lib/cli/base.js

```diff
@@ -1,5 +1,6 @@
 import chalk from "chalk";
 import {isLogLevelEnabled} from "@ui5/logger";
+import ConsoleWriter from "@ui5/logger/writers/Console";
```

Comment:
should we wrap such an event in the index.js of the logger so we can hide the ConsoleWriter

Reply:
Not sure, I guess that depends on whether it would be relevant for other writers in the future too.

As of today, it's only relevant for the ConsoleWriter. So maybe we should not wrap it additionally?

Reply:
I see.... writers/Console is public so I am fine with it